src/GrumPHP/Console/Application.php

```diff
@@ -53,6 +53,26 @@ protected function getDefaultInputDefinition()
     }
 
     /**
+     * @return string
+     */
+    protected function getDefaultConfigName()
```

Comment:
It seems like this value will be converted automatically to absolute when it will be used with commands: https://github.com/phpro/grumphp/blob/master/src/GrumPHP/Console/Application.php#L121
Right or I missed something? (I'm not sure)

In the other hand, relative path in command output (when you run just `grumphp` or `grumphp list`) is more readable.

Function name `getConfigDefaultPath` even better than `getDefaultConfigPath` (exact as extra key named).

WDYT?